scripts/helpers/dlc-admin-safes.js

```diff
@@ -15,4 +15,8 @@ module.exports = DLCAdminSafeAddresses = {
         medium: '0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266',
         critical: '0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266',
     }, // Hardhat default deployer account
+    mainnet: {
+        medium: '0x9506Ea24038609679732855F757041a3C1C06623', // NOTE: temporary until mainnet setup
+        critical: '0x9506Ea24038609679732855F757041a3C1C06623',
+    },
```

Comment:
do we want these to be the same? I guess this is a "mainnet" level system, and so i guess it should match the security design of arb.